examples/notebooks/Validating_mechanical_models_Enertech_DFN.ipynb

```diff
@@ -0,0 +1,305 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model = pybamm.lithium_ion.DFN(\n",
+    "    build=False,options = {\n",
+    "        \"particle\": \"Fickian diffusion\", \n",
+    "        \"cell geometry\": \"arbitrary\", \n",
+    "        \"thermal\": \"lumped\", \n",
+    "        \"particle cracking\": \"no cracking\",\n",
+    "    }\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can get the parameter set `Ai2020` for the model, which includes the mechanical properties required by the mechanical model."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# param = model.default_parameter_values\n",
+    "chemistry = pybamm.parameter_sets.Ai2020\n",
+    "\n",
+    "# # Below commands are used to add the mechanical parameters missing from other parameter sets \n",
+    "# import pandas as pd\n",
+    "# mechanics = pd.read_csv(\"pybamm/input/parameters/lithium-ion/mechanicals/lico2_graphite_Ai2020/parameters.csv\", \n",
+    "#                        index_col=0, comment=\"#\", skip_blank_lines=True, header=None)[1][1:].dropna().astype(float).to_dict()\n",
+    "# param.update(mechanics, check_already_exists=False)\n",
+    "param = pybamm.ParameterValues(chemistry=chemistry)\n",
+    "experiment05C = pybamm.Experiment([\"Discharge at 0.5C until 3 V\"])\n",
+    "experiment1C = pybamm.Experiment([\"Discharge at 1C until 3 V\"])\n",
+    "experiment2C = pybamm.Experiment([\"Discharge at 2C until 3 V\"])\n",
+    "var = pybamm.standard_spatial_vars\n",
+    "var_pts = {\n",
+    "  var.x_n: 50,\n",
+    "  var.x_s: 50,\n",
+    "  var.x_p: 50,\n",
+    "  var.r_n: 20,\n",
+    "  var.r_p: 20,\n",
+    "}\n",
+    "\n",
+    "experiment = [experiment05C, experiment1C, experiment2C]\n",
+    "solution = []\n",
+    "\n",
+    "for ep in experiment:\n",
+    "    sim = pybamm.Simulation(\n",
+    "        model,\n",
+    "        var_pts=var_pts,\n",
+    "        experiment=ep,\n",
+    "        parameter_values=param,\n",
+    "        solver=pybamm.CasadiSolver(dt_max=0.001)\n",
+    "    )\n",
+    "    solution.append(sim.solve())\n",
+    "\n",
+    "\n",
+    "solution05C, solution1C, solution2C = solution"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Load experimental results of the Enertech cells, Ref. Ai et al. JES 2020\n",
+    "https://iopscience.iop.org/article/10.1149/2.0122001JES"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# load experimental results\n",
+    "import pandas as pd\n",
+    "path = \"pybamm/input/discharge_data/Enertech_cells/\"\n",
```

Comment:
this data is not in the commit

Reply:
Strange here. Does Git skip `txt` files? Will try another way to upload those data.

Reply:
Yes, git skips the text files because we have `*.txt` in `.gitignore` (there was a reason but I don't remember why). You can add them with `git add -f filename` or something along those lines